src/elementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -1,30 +1,80 @@
-import React from 'react'
+import React, { useEffect, useState } from 'react'
 import { ErrorBoundary, pluginProvider } from './'
 import { ApplicationViewWrapperProps, PluginComponents } from './types'
 import evaluateExpression from '@openmsupply/expression-evaluator'
 import { useUpdateResponseMutation } from '../utils/generated/graphql'
+import { ResponseFull } from '../utils/types'
 
 const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) => {
   const {
-    templateElement: { elementTypePluginCode: pluginCode },
+    templateElement,
     isVisible,
     isEditable,
     isRequired,
     currentResponse,
+    allResponses,
   } = props
 
-  if (!pluginCode || !isVisible) return null
+  const { elementTypePluginCode: pluginCode } = templateElement
+  const { validation: validationExpression, validationMessage } = templateElement?.parameters
 
   const [responseMutation] = useUpdateResponseMutation()
+  const [pluginMethods, setPluginMethods] = useState({
+    validate: (validationExpress: any, validationMessage: any, evaluator: any) =>
+      console.log('notLoaded'),
+  })
```

Comment:
I'm not sure exactly what you mean, perhaps you can show me tomorrow. 

Sorry -- some of this code is @andreievg 's since we did it together, so I'm not entirely clear about some of his syntax.